tests/test_meshtastic_utils_connect_paths.py

```
@@ -1,4 +1,5 @@
 import time
+from typing import Any, Callable
```

Ruff (0.15.9)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)